src/lib/authserver/ui/new_dir_inst.rb

```diff
@@ -120,7 +120,7 @@ def ok_handler
     # Always remove the ini file
     DS389.remove_setup_ini
     if !ok
-      Popup.Error(_('Failed to set up new instance! Log output may be found in /var/log/YaST/y2log'))
+      Popup.Error(_('Failed to set up new instance! Log output may be found in /var/log/YaST2/y2log'))
```

Comment:
You could interpolate [`Yast::Directory.logdir`](https://github.com/yast/yast-yast2/blob/7202cd8968a946879ea4df0ae071049d4f46f179/library/general/src/modules/Directory.rb#L130) instead

```suggestion
      Popup.Error(
         # TRANSLATORS: %{y2log_path} is replaced by the YaST2 logs path.
         format(
          _('Failed to set up new instance! Log output may be found in %{y2log_path}'),
          y2logpath: Yast::Directory.logdir
        )
      )
```

Reply:
Thanks, I implemented your suggestion.